charts/dv-pod/values.yaml

```
@@ -325,20 +375,19 @@ charon:
 
   # --- Charon application internal configuration (translates to CLI flags) ---
   config:
-    # -- Path within EACH Charon container where the SHARED ENR private key file will be mounted.
-    privateKeyFile: "/data/charon_shared_enr_private_key"
+    # -- Path within the Charon container where the ENR private key file will be mounted.
+    privateKeyFile: "/data/charon-enr-private-key"
 
 # -- Kubernetes secrets names that might be used as suffixes or for other purposes.
-# For the shared ENR, the secret name is either defined in 'charon.enr.existingSecret.name' 
-# or generated by the ENR job (e.g., {{ .Release.Name }}-charon-enr-key).
-# The 'enrPrivateKey' field below is NOT directly used for the shared ENR secret name in this model.
+# For the ENR, the secret name is either defined in 'charon.enr.existingSecret.name' 
+# or generated by the ENR job (e.g., {{ .Release.Name }}-dv-pod-enr-key).
 secrets:
-  # -- Suffix or name component for validator keys secrets (if applicable, distinct from ENR)
-  validatorKeys: "validator-keys"
-  # -- This field is less relevant for the SHARED ENR model. Retained for other potential uses or legacy compatibility.
-  # For shared ENR, see 'charon.enr' section above.
-  enrPrivateKey: "charon-enr-private-key" # Suffix if per-pod ENRs were used; less relevant now for shared. 
+  # -- Suffix for ENR private key secret (used internally by templates)
+  enrPrivateKey: "charon-enr-private-key"
   # -- Name or suffix for the charon cluster lock secret
+  # To skip the DKG process entirely, create a Kubernetes secret containing your cluster-lock.json:
+  # kubectl create secret generic cluster-lock --from-file=cluster-lock.json
+  # The DKG sidecar will detect this file and exit immediately, allowing Charon to start with the existing lock.
   clusterlock: "cluster-lock"
 
```

```suggestion
  # kubectl create secret generic cluster-lock --from-file=cluster-lock.json
  # The DKG sidecar will detect this file and exit immediately, allowing Charon to start with the existing lock.
  clusterLock: "cluster-lock"

```

Cluster locks can be a couple megabytes in size, won't that be too big for a secret? we may want to make this a configmap